pages/index.vue

```diff
@@ -1,53 +1,125 @@
 
 <template>
-<div class="w-100 h-100">
-  <div class="center">
+<div class="w-100 h-100 d-flex flex-column  align-items-center justify-content-between">
+  <div class="position-absolute  top-0 w-75 mt-3  text-white">
+   
+      <div class="ck-content  p-3 text-white">
+                <div ref="contentEl" class="cont" v-html="content"></div>
+      </div>
+                        
+  </div>
+  <div class="center ">
     <div class="d-flex justify-content-between">
-
-      <NuxtLink to="/repository" class="text-decoration-none">
-        <div class="card card-bg mx-5" >
-          <div class="card-body d-flex align-items-center justify-content-center">
-            <h5 class="card-title text-center card-title-white text-decoration-none">
-              {{t('Repository')}}
-            </h5>
-          </div>
-        </div>
-      </NuxtLink>
-      <!-- <div class="card card-sep mx-5" >
-          <div class="card-body d-flex align-items-center justify-content-center">
-            &nbsp;
-
-          </div>
-      </div> -->
-      <NuxtLink to="/ism" class="text-decoration-none">
-        <div class="card card-bg mx-5" >
-          <div class="card-body d-flex align-items-center justify-content-center">
-            <h5 class="card-title text-center card-title-white text-decoration-none">
-              {{t('Information Sharing Mechanism')}}
-            </h5>
-          </div>
-        </div>
+      <NuxtLink  v-for="(button, key) of buttons"  :to="localPath(button.path)" class="text-decoration-none">
+        <button class="landing-button mx-5" :class="{ 'landing-button-alt': key%2 }" >
+          <span class="text-center">{{button.name}}</span>
+        </button>
       </NuxtLink>
     </div>
   </div>
+
+  <ClientOnly>
+    <div v-if="canEdit" class="position-absolute top-0 w-100 d-flex justify-content-end">
+   
+      <button  @click="gotToEdit" class="btn  btn-sm btn-light m-2" external>
+          <Icon name="edit" color="#000000"/>
+      </button>
+
+    </div>
+  </ClientOnly>
 </div>
 </template>
 
 
 <script setup >
 definePageMeta({ layout: 'landing' });
 
-const { t }  = useI18n();
 
+const { isAuthenticated, hasRole } = useScbdAuth     ();
+const { oasisApi        } = useRuntimeConfig().public;
+const   url               = useRequestURL   ();
+const   localPath         = useLocalePath();
+const   page              = useState('page');
+const { t, locale }       = useI18n();
+const   contentEl         = ref();
+const   _id               = computed(()=>page.value?._id);
+const   content           = computed(()=>page.value?.content);
```

Comment:
make sure you DOMPurify the HTML before passing it ot `v-html`
smaple here https://github.com/scbd/www.cbd.int/blob/master/app/services/html.js#L8